### PR TITLE
Create Should/Warning Test Files

### PR DESCRIPTION
**Pull request by amy-silcock**

### CODE REVIEW

**Comment by amy-silcock** [comment]
I'll add in all the additional test files here and let you know when they're ready for review.

**Comment by amy-silcock** [comment]
@rolfkleef test files are ready for review.

File: tests/iati/testfiles/1.3.13.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
  <iati-activity xml:lang="en" default-currency="USD" last-updated-datetime="2014-09-10T07:15:37Z" humanitarian="1" linked-data-uri="http://data.example.org/123456789" hierarchy="1" budget-not-provided="1">
 
   <!--iati-identifier starts-->
-  <iati-identifier>AA-AAA-123456789&ABC123/</iati-identifier>
+  <iati-identifier>AA-AAA-123456789&amp;ABC123/</iati-identifier>
```
Comment (amy-silcock) [inline tests/iati/testfiles/1.3.13.xml:11]: Hi @rolfkleef, on this one users will likely be submitting & into their ids, and not the ascii version. Why is this needed in the test file?

And if it's needed for this files then is it also needed for other test files checking for non legit characters?

Comment (rolfkleef) [reply tests/iati/testfiles/1.3.13.xml:11]: just writing "&" is illegal XML, any tool that writes XML will write an ampersand out as "&amp;" in the XML code

I fixed these characters in earlier test files as well, and added an extra test step to make sure test files are valid XML
